docs/guides/open_telemetry.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: otel
+title: OpenTelemetry
+description: How to instrument crawlers with OpenTelemetry
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import CodeBlock from '@theme/CodeBlock';
+
+import InstrumentCrawler from '!!raw-loader!./code_examples/open_telemetry/instrument_crawler.py';
+
+
+[OpenTelemtery](https://opentelemetry.io/) is a collection of APIs, SDKs, and tools to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) to help you analyze your software’s performance and behavior. In context of crawler development it can be used to better understand how the crawler internally works, identify bottlenecks, debug, log metrics, and more.
+
+In this guide it will be shown how to set up OpenTelemetry and instrument specific crawler to see traces of individual requests that are being processed by the crawler. OpenTelemetry on its own does not provide out of the box tool for convenient visualisation of the exported data (apart from printing to the console), but there are several good available tools to do that. In this guide we will use [Jaeger](https://www.jaegertracing.io/) to visualise the telemetry data.
+
+
+## Set up the Jaeger
+This guide will show how to set up the environment locally to run the example code and visualize the telemetry data in Jaeger that will be running locally in a [docker](https://www.docker.com/) container.
+
+To start the preconfigured docker container you can use following command:
+
+```bash
+docker run -d --name jaeger -e COLLECTOR_OTLP_ENABLED=true -p 16686:16686 -p 4317:4317 -p 4318:4318 jaegertracing/all-in-one:latest
+```
+For more details about the Jaeger setup see [getting started](https://www.jaegertracing.io/docs/2.7/getting-started/) section in their documentation.
+You can see the Jaeger UI in your browser by navigating to the http://localhost:16686
+
+## Instrument the Crawler
+Now you can proceed with instrumenting the crawler to send the telemetry data to the Jaeger and running it. To have the Python environment ready, you should install either **crawlee[all]** or **crawlee[otel]**, this will ensure that OpenTelemetry dependencies are installed, and you can run the example code snippet.
+In the following example you can see the function `instrument_crawler` that contains the instrumentation setup and is called before the crawler is started. If you have already set up the Jaeger, then you can just run following code snippet.
+
+<CodeBlock className="language-python">
+    {InstrumentCrawler}
+</CodeBlock>
+
+
+## Analyze the results
```

Comment:
I would appreciate more information here as well - perhaps an overview of web-based UIs for open telemetry and this kind of analysis, followed by a deeper look at the Jaeger UI.